index.js

```diff
@@ -1,17 +1,17 @@
-function importModule(requiredModule){
+function importModule(requiredModule) {
     return requiredModule && requiredModule.default || requiredModule;
 }
 
 function initDOMPurifyWithJSDOM() {
   const DOMPurifyInitializer = importModule(require('dompurify'));
   const { JSDOM } = importModule(require('jsdom'));
   const { window } = new JSDOM('<!DOCTYPE html>');
-  return DOMPurifyInitializer(window)
+  return DOMPurifyInitializer(window);
 }
 
-function resolveDOMPurify(){
+function resolveDOMPurify() {
   const isClientSide = typeof process === 'undefined';
-  isClientSide ? importModule(require('dompurify')) : initDOMPurifyWithJSDOM();
+  return isClientSide ? importModule(require('dompurify')) : initDOMPurifyWithJSDOM();
```

Comment:
@NicholasEllul I've just added the return statement to fix tests.

Reply:
Haha this is how you can tell i've been writing too much ruby lately, thanks

Reply:
Ah, yes, Ruby automatically returns the result of the last statement.